app/index.js

```diff
@@ -303,7 +303,7 @@ KeystoneGenerator.prototype.tasks = function tasks () {
 
 		var gruntFiles = ['concurrent', 'express', 'node-inspector', 'nodemon'];
 
-		this.template('_Gruntfile.js', 'Gruntfile.js');
+		this.template('_gruntfile.js', 'gruntfile.js');
```

Comment:
I changed the file case in my repo ignore that fix but the if else statement definitely needs to be merged